BibBuddy/app/src/main/java/de/bibbuddy/NoteDao.java

```diff
@@ -364,9 +349,9 @@ public Long findBookIdByNoteId(Long noteId) {
         + " n ON (n." + DatabaseHelper._ID + " = lnk." + DatabaseHelper.NOTE_ID + ")"
         + " WHERE n." + DatabaseHelper.TYPE + " = ? AND lnk." + DatabaseHelper.BOOK_ID + "= ?";
 
-    Cursor cursor = db.rawQuery(selectQuery, new String[] {String.valueOf(NoteTypeLut.TEXT.getId()),
-        String.valueOf(bookId)
-        });
+    Cursor cursor = db.rawQuery(selectQuery, new String[] {
+        String.valueOf(NoteTypeLut.TEXT.getId()),
+        String.valueOf(bookId) });
```

Comment:
```suggestion
        String.valueOf(bookId) 
      });
```

Reply:
This suggestion gives me a checkstyle error.

Reply:
ok then just use your version